tools/docker/README.md

```diff
@@ -18,38 +18,35 @@ _**All commands mentioned in this document should be run from the base Jetpack d
 
 ### Prerequisites
 
-* [Docker](https://hub.docker.com/search/?type=edition&offering=community)
-* [NodeJS](https://nodejs.org)
-* [Pnpm](https://pnpm.io/)
-* Optionally [Ngrok](https://ngrok.com) client and account or some other service for creating a local HTTP tunnel. It’s fine to stay on the free pricing tier with Ngrok.
+To use Jetpack's Docker environment, you will need:
 
-Install prerequisites; you will need to open up Docker to install its dependencies.
+- A [local copy of the Jetpack repository](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md##clone-the-repository).
```

Comment:
There is a double `#` (##clone-the-repository) in the anchor of the URL. Which is not pointing to the correct section on the destination page.

Reply:
Good catch. 1421c21659220bb902e5e6955ba3bc2583297e9e